ibm/conns/config.go

```diff
@@ -1149,8 +1144,6 @@ func (c *Config) ClientSession() (interface{}, error) {
 			if err != nil {
 				session.bmxUserFetchErr = fmt.Errorf("[ERROR] Error occured while fetching auth key for account user details: %q", err)
 				session.functionConfigErr = fmt.Errorf("[ERROR] Error occured while fetching auth key for function: %q", err)
-				session.powerConfigErr = fmt.Errorf("[ERROR] Error occured while fetching the auth key for power iaas: %q", err)
-				session.ibmpiConfigErr = fmt.Errorf("[ERROR] Error occured while fetching the auth key for power iaas: %q", err)
```

Comment:
why is line no. 1153 removed?

Reply:
Power API will authenticate users via Authenticator hence we don't need right?